animations.md

```diff
@@ -17,27 +17,34 @@ functions directly by reading their current value and listening to their
 state changes or they can use the animations as the basis of more elaborate
 animations that they pass along to other widgets.
 
-## Animation
+## Key animation classes
+
+This section describes some of the common APIs you will use for creating
+animation effects and controlling how they run.
+
+### Animation
 
 The primary building block of the animation system is the
 [`Animation`](http://docs.flutter.io/flutter/animation/Animation-class.html)
-class.  An animation represents a value of a specific type that can change
+class. An animation represents a value of a specific type that can change
 over the lifetime of the animation. Most widgets that perform an animation
-receive an `Animation` object as a parameter, from which they read the current
-value of the animation and to which they listen for changes to that value.
+receive an [`Animation`](http://docs.flutter.io/flutter/animation/Animation-class.html)
+object as a parameter, from which they read the current value of the animation
+and to which they listen for changes to that value.
 
-### `addListener`
+Two key methods of this class are:
 
-Whenever the animation's value changes, the animation notifies all the
-listeners added with
++ **[`addListener()`](http://docs.flutter.io/flutter/animation/Animation/addListener.html):**
+Whenever the animation's value changes, the animation
+notifies all the listeners added with
 [`addListener`](http://docs.flutter.io/flutter/animation/Animation/addListener.html).
 Typically, a [`State`](http://docs.flutter.io/flutter/widgets/State-class.html)
 object that listens to an animation will call
 [`setState`](http://docs.flutter.io/flutter/widgets/State/setState.html) on
 itself in its listener callback to notify the widget system that it needs to
 rebuild with the new value of the animation.
 
-This pattern is so common that there are two widgets that help widgets rebuild
+     This pattern is so common that there are two widgets that help widgets rebuild
```

Comment:
This is intended markdown formatting to keep the new paragraph part of the same indent block as its antecedent.